components/Accordion/Accordion.vue

```diff
@@ -79,5 +86,10 @@ const iconColor = computed(() => {
       return 'text-neutral-500'
   }
 })
-onUnmounted(() => unregister(accordionId))
+onMounted(() => {
+  if (route.hash === `#${titleAccordionId}`) {
+    open(titleAccordionId)
```

Comment:
I think it could be cool to have better ID in the hash, maybe set custom IDs for the FAQ?

Reply:
I wanted to keep the addition of new accordion simple, so I relied on generated ids but we can use custom ids instead